api/lib/src/lib.rs

```diff
@@ -356,6 +361,42 @@ pub trait BrokerApi: SignedExtrinsicApi {
 			bail!("No SwapDepositAddressReady event was found");
 		}
 	}
+	async fn withdraw_fees(
+		&self,
+		asset: Asset,
+		destination_address: EncodedAddress,
+	) -> Result<WithdrawFeesDetail> {
+		let (tx_hash, events, ..) = self
+			.submit_signed_extrinsic(RuntimeCall::from(pallet_cf_swapping::Call::withdraw {
+				asset,
+				destination_address,
+			}))
+			.await
+			.until_in_block()
+			.await
+			.context("Request to withdraw broker fee for ${asset} failed.")?;
+
+		if let Some(state_chain_runtime::RuntimeEvent::Swapping(
+			pallet_cf_swapping::Event::WithdrawalRequested {
+				egress_amount: _,
+				egress_fee: _,
+				destination_address: _,
```

Comment:
Why not return all of the information? All of this is interesting to the broker.